easybuild/easyblocks/o/openbabel.py

```diff
@@ -8,7 +8,7 @@
 # Flemish Research Foundation (FWO) (http://www.fwo.be/en)
 # and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
 #
-# https://github.com/easybuilders/easybuild
+# http://github.com/hpcugent/easybuild
```

Comment:
@ostueker This is wrong after the move to the `easybuilders` organisation, please change it back?